unit/java_bytecode/java_string_library_preprocess/convert_exprt_to_string_exprt.cpp

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************\
+
+ Module: Java string library preprocess.
+         Test for converting an expression to a string expression.
+
+ Author: DiffBlue Limited. All rights reserved.
+
+\*******************************************************************/
+
+#include <catch.hpp>
+#include <util/expr.h>
+#include <util/std_code.h>
+#define UNIT_TEST
```

Comment:
See my comment above.

Reply:
Once this is renamed CBMC_UNIT_TEST or similar, define it globally (or perhaps add it to CXXFLAGS in the Makefile?) so there's a uniform method for people wanting to pull similar pseudo-friend trickery to test other classes.